priv/gettext/en/LC_MESSAGES/errors.po

```diff
@@ -2,7 +2,7 @@ msgid "can't be blank"
 msgstr "This field can't be blank."
 
 msgid "has already been taken"
-msgstr "This email address already exists."
+msgstr "This value should be unique."
```

Comment:
Is there not a way to make the distinction between emails (in this case) and other uniqueness violations?

Reply:
It's possible but we gonna need to call a different translation. Opened an issue for identifying and changing on all places of the app to have a specific message for email: https://github.com/OpenFn/lightning/issues/2824